### PR TITLE
Update category-porn

### PR DESCRIPTION
**Pull request by bijoni**

### CODE REVIEW

**Review by rootmelo92118 — APPROVED** [review]

File: data/category-porn [hunk 1]
```diff
@@ -5794,9 +5890,13 @@ regexp:(^|\.)spankwire[1-9]\.com$
 regexp:(^|\.)syjc1[1-6]\.xyz$
 regexp:(^|\.)tianbiav([1-9]|1[0-6])?\.com$
 regexp:(^|\.)tqav[1-9][0-9]\.com$
+regexp:(^|\.)tt[1-2][0-9]\.tv$
 regexp:(^|\.)ttghg[1-9][0-9]\.xyz$
+regexp:(^|\.)tttv[1-100]\.com$
 regexp:(^|\.)twseb([1-9][0-9]?)?\.com$
 regexp:(^|\.)uu[a-z][1-9][0-9]?\.com$
+regexp:(^|\.)wmm33[0-5]\.com$
+regexp:(^|\.)whtdh0[1-3]\.cc$
```
Comment (IceCodeNew) [inline data/category-porn:5899]: sort

File: data/category-porn [hunk 2]
```diff
@@ -5794,9 +5890,13 @@ regexp:(^|\.)spankwire[1-9]\.com$
 regexp:(^|\.)syjc1[1-6]\.xyz$
 regexp:(^|\.)tianbiav([1-9]|1[0-6])?\.com$
 regexp:(^|\.)tqav[1-9][0-9]\.com$
+regexp:(^|\.)tt[1-2][0-9]\.tv$
 regexp:(^|\.)ttghg[1-9][0-9]\.xyz$
+regexp:(^|\.)tttv[1-100]\.com$
```
Comment (IceCodeNew) [inline data/category-porn:5895]: It does not look right to me

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

**Comment by IceCodeNew** [comment]
@rootmelo92118 要检查得仔细一点哦。

File: data/category-porn (hunk 1 above)
Comment (bijoni) [reply data/category-porn]: Sorted

**Review by bijoni — COMMENTED** [review]
Amended the sorting

File: data/category-porn (hunk 2 above)
Comment (bijoni) [reply data/category-porn:5895]: Verified with no problem.

Comment (rootmelo92118) [reply data/category-porn:5895]: > Verified with no problem.

Could you provide the original domains?

Comment (bijoni) [reply data/category-porn:5895]: From tttv1.com through tttv100.com

Comment (rootmelo92118) [reply data/category-porn:5895]: > From tttv1.com through tttv100.com

Do you know it will only match tttv1.com and tttv0.com ?

Comment (rootmelo92118) [reply data/category-porn:5895]: According to rule `tttv+1-100任意数.com` on the home page in [tttv1.com](tttv1.com), I rewrote a new pattern
```
(^|\.)tttv([1-9]|[1-9][0-9]|100)\.com$
```
What do you think? @IceCodeNew @bijoni

Comment (rootmelo92118) [reply data/category-porn]: @IceCodeNew Solved.

Comment (IceCodeNew) [reply data/category-porn]: `[1-100]` does not make sense.

**Review by IceCodeNew — APPROVED** [review]

**Pull request merged**